network-api/networkapi/campaign/models.py

```diff
@@ -85,13 +85,33 @@ class Petition(models.Model):
     share_link = models.URLField(
         max_length=1024,
         help_text="Link that will be put in share button",
-        blank=True
+        blank=True,
+        editable=False
     )
 
     share_link_text = models.CharField(
         max_length=20,
         help_text="Text content of the share button",
         default="Share this",
+        blank=True,
+        editable=False
+    )
+
+    share_twitter = models.CharField(
+        max_length=20,
```

Comment:
check limit

Reply:
Limit is 9, and it roughly looks like this "sp_185606" I'm happy to leave it at 20 tbh, but if you want me to tighten it up to 9 I can do that too.